docs/test-suite-design.md

```diff
@@ -32,6 +32,13 @@ For example, tests in HTML for ["The History
 interface"](https://html.spec.whatwg.org/multipage/history.html#the-history-interface)
 are located in `html/browsers/history/the-history-interface/`.
 
+Many directories also include a file named `META.yml`. This file may define any
+of the following properties:
+
+- `spec` - a link to the specification that the tests in the directory exercise
```

Comment:
I somewhat dislike this use of the word "exercise"—it seems just obscure enough to be a bit confusing; or maybe that's just the grammar here with it following the object?

Reply:
Yeah, I think `specification exercised by the tests in the directory` is fine, although `covered` would be even better.